tests/samples/test_target_sqlite.py

```diff
@@ -84,8 +85,10 @@ def test_sync_sqlite_to_sqlite(
     )
     orig_stdout.seek(0)
     tapped_config = dict(sqlite_sample_target.config)
-    catalog = _get_tap_catalog(SQLiteTap, config=tapped_config, select_all=True)
-    tapped_target = SQLiteTap(config=tapped_config, catalog=catalog)
+    tapped_target = SQLiteTap(
+        config=tapped_config,
+        catalog=sqlite_sample_db_catalog.to_dict(),
+    )
```

Comment:
The data tapped from the loaded SQLite db had a different catalog, i.e. streams that are incremental during the first sync, are "full table" in the downstream sync, so the message count was different and the test was failing.